spec/linter-codeclimate-spec.js

```diff
@@ -13,14 +14,15 @@ describe('The codeclimate provider for Linter', () => {
 
     waitsForPromise(() =>
       Promise.all([
-        atom.packages.activatePackage('language-ruby'),
         atom.packages.activatePackage('linter-codeclimate'),
       ]),
     );
   });
 
   it('works with a valid .codeclimate.yml file', () =>
-    waitsForPromise(() =>
+    waitsForPromise(
+      { timeout: 10000 },
```

Comment:
The 10 second dependency is determined in Atom, since it's what sets up and runs the test environment.